style_editor.html

```diff
@@ -8830,6 +8878,42 @@
 var use_graflex = true;
 var bad_fps = 0;
 
+</script>
+
+<div id="popup_window" class="popup-window">
+  <p id="popup_message"></p>
+  <button title="OK" onclick="dismissPopupMessage()">OK</button>
+  <br>
+  <label title="Disable this popup again. Refresh page to show again.">
```

Comment:
"Disable this popup again"
What does that mean?

Reply:
bad edit. used to be "Don't show this popup again."
Then added the Refresh part, changed "Don't show" to "Disabled", but left the "again."